src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    
+    /**
+     * Returns String feedbackToUser
+     */
+    public String getFeedbackToUser(){
+    	return feedbackToUser;
+    }
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

If you have commits for different activities on the same branch, create a new branch and use cherry-pick to choose which commits to include.